.github/workflows/renovate-bump-chart-version.yaml

```diff
@@ -0,0 +1,92 @@
+name: Renovate Bump Chart Version
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - 'charts/**/*'
+
+jobs:
+  renovate-bump-chart-version:
+    name: Renovate Bump Chart Version
+    runs-on: ubuntu-latest
+    if: github.actor == 'renovate[bot]'
+    permissions:
+      contents: write
```

Comment:
The logic looks good, I'm not sure if renovate will actually have permissions to write. We'll have to see 🤞

Reply:
thats not how it works.

The if condition means, run the job only, if the PR comes from renovate. Inside the github action, the action user will still do things, not renovate.

Reply:
You are right, I was mistaking it with dependabot permissions that are special ([docs](https://docs.github.com/en/actions/security-for-github-actions/security-guides/automatic-token-authentication#permissions-for-the-github_token)):
> Workflow runs triggered by Dependabot pull requests run as if they are from a forked repository, and therefore use a read-only GITHUB_TOKEN. These workflow runs cannot access any secrets. For information about strategies to keep these workflows secure, see [Security hardening for GitHub Actions](https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions).